Userland/Libraries/LibC/sys/ioctl_numbers.h

```diff
@@ -74,6 +74,35 @@ struct FBFlushRects {
     struct FBRect const* rects;
 };
 
+struct VirGL3DResourceSpec {
+    unsigned target;
+    unsigned format;
+    unsigned bind;
+    unsigned width;
+    unsigned height;
+    unsigned depth;
+    unsigned array_size;
+    unsigned last_level;
+    unsigned nr_samples;
+    unsigned flags;
+    unsigned created_resource_id;
```

Comment:
Use u32, if the size is fixed
same below

Reply:
I don't believe u32 is usable here, that is defined in AK/Types.h, which is a C++ header file. `LibC/sys/ioctl_numbers.h` is part of libc, so the header needs to be readable by a C compiler, which means we can't include `AK/Types.h`.

Reply:
@Hendiadyoin1 Unless you think we should also define u32 somewhere where libC can use it?

Reply:
I'll assume not, then.